pandas/plotting/_core.py

```diff
@@ -82,26 +92,28 @@ def hist_series(
         yrot=yrot,
         figsize=figsize,
         bins=bins,
+        legend=legend,
         **kwargs,
     )
 
 
 def hist_frame(
-    data,
-    column=None,
+    data: "DataFrame",
+    column: Optional[Union[str, Sequence[str]]] = None,
```

Comment:
```suggestion
    column: Union[Label, Sequence[Label]] = None,
```

Can import from pandas._typing